### PR TITLE
feat(ledger): make ledger creation manual

### PR DESCRIPTION
**Pull request by gfyrag**
Fixes ENG-235

### CODE REVIEW

File: components/ledger/internal/api/v2/routes.go [hunk 1]
```diff
@@ -37,7 +37,8 @@ func NewRouter(
 		router.Get("/_info", getInfo(b))
 
 		router.Route("/{ledger}", func(router chi.Router) {
-			router.Post("/", configureLedger(b))
+			router.Post("/", createLedger(b))
+			router.Get("/", getLedger(b))
```
Comment (paul-nicolas) [inline components/ledger/internal/api/v2/routes.go:41]: Do we want a list also to display them on console ?

File: components/ledger/internal/engine/resolver.go [hunk 2]
```diff
@@ -67,6 +69,15 @@ func NewResolver(storageDriver *driver.Driver, options ...option) *Resolver {
 	return r
 }
 
+func (r *Resolver) startLedgerUsingStore(ctx context.Context, name string, store *ledgerstore.Store) *Ledger {
```
Comment (paul-nicolas) [inline components/ledger/internal/engine/resolver.go:72]: start is called in the getLedger and also in the create ? Should we have a way to know if the ledger is already started or not in order to not launch the same goroutine ?

Comment (paul-nicolas) [reply components/ledger/internal/engine/resolver.go:72]: My bad, it's not launch in the get since you search in the map before, discard this comment

**Review by paul-nicolas — APPROVED** [review]

File: components/fctl/cmd/ledger/create.go [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+	if err != nil {
+		return nil, err
+	}
+
+	stack, err := fctl.ResolveStack(cmd, cfg, organizationID)
+	if err != nil {
+		return nil, err
+	}
+
+	if !fctl.CheckStackApprobation(cmd, stack, "You are about to create a new transaction") {
```
Comment (paul-nicolas) [inline components/fctl/cmd/ledger/create.go:73]: A new transaction ?

Comment (gfyrag) [reply components/fctl/cmd/ledger/create.go:73]: oups :D

**Review by paul-nicolas — APPROVED** [review]

**Pull request merged**